src/commands/Minion/bury.ts

```diff
@@ -0,0 +1,117 @@
+import { CommandStore, KlasaMessage } from 'klasa';
```

Comment:
I think we need to rename bury cmd to bones, or make bury cmd only for burying, just for naming